week2/class.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+        <script>
+    var names = ["Tim", "Timmy", "Timy", "Timm", "Timantha", "Timbo", "Timmey", "Timmie", "Timey", "Timie", "Timmbo", "Timi", "Timmi", "Timei", "Timmei"];
+    var students = 15;
+    var homework = 5;
+    var quiz = 3;
+    var exams = 2;
+
+
+  function table(){
+
+    var array = [];
+    var x = 0;
+    var rand = 0;
+    var name = "Tim";
+    var haverage = 0;
+    var eaverage = 0;
+    var qaverage = 0;
+    var average = 0;
+    var hgrades = [];
+    var qgrades = [];
+    var egrades = [];
+
+
+      rand = Math.floor(Math.random * 14);
+      name = names[rand];
+
+      while (x < homework){
+      rand = Math.floor(Math.random * 5);
+
+      if(rand >= 3){
+
+      rand = Math.floor(Math.random * 100);
```

Comment:
this is some really really really bad indentation. It makes the code very very hard to read.